src/cozmo/run.py

```diff
@@ -286,6 +289,10 @@ def _remove_forward(self, serial):
             except:
```

Comment:
Using an `except` branch here for `SDKVersionMismatch` which then `raise`s that error might be cleaner than using the `version_mismatch` variable

Reply:
I want it to try each Android in turn, and only raise SDKVersionMismatch if they all fail